handler.go

```diff
@@ -29,16 +36,24 @@ func (f *Faucet) defaultHTTPHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	// Extract the requests
-	requests, err := extractRequests(requestBody)
-	if err != nil {
-		http.Error(
-			w,
-			"invalid request body",
-			http.StatusBadRequest,
-		)
+	var requests Requests
 
-		return
+	if to := r.FormValue("toaddr"); to != "" {
+		requests = Requests{
+			{To: to},
+		}
```

Comment:
I do not like the fact we are modelling the behavior of a generic TM2 faucet based on `gnoweb` (and exact field names in `gnoweb`. The faucet is envisioned to work through POST requests of a specific format (or we can define our own handlers, endpoints, the faucet lib allows for that).

We are deprecating the old gno faucet anyways, so might as well make the change now.

I propose we take one of these routes:
- modify `gnoweb` to send a POST with a standard body that is expected in this faucet implementation, this way only 1 thing is changed, and it's on the Gnoweb side (easy route, and longterm the final version, since we are deprecating the old faucet)
- use this faucet as a lib in gno, and extend the faucet to have specific handlers / routes you would use for gnoweb (define custom handlers, like we did for gnochess). This option is less favorable, because it just introduces complexity we'd drop later on when deleting the old faucet

Reply:
100% ok with this, i just choose a quicker solution for having a fix to work with

Reply:
but the "form" system is an easier system for the way we're using gno/templating html